sycl/test/bit_cast/bit_cast.cpp

```diff
@@ -0,0 +1,91 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t1.out
+// RUN: env SYCL_DEVICE_TYPE=HOST %t1.out
+// RUN: %CPU_RUN_PLACEHOLDER %t1.out
+// RUN: %GPU_RUN_PLACEHOLDER %t1.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
```

Comment:
```suggestion
// RUN: %ACC_RUN_PLACEHOLDER %t1.out
```

Or, which is better, IMHO, you could rename `t1` to `t` everywhere else

Reply:
Thanks, fixed.